__tests__/app.test.js

```diff
@@ -361,5 +361,38 @@ describe('PATCH: /api/articles/:article_id', () => {
             expect(body.msg).toBe('bad request')
     })
     })
-
+})
+describe('DELETE: /api/comments/:comment_id', () => {
+    test('DELETE 200:removes comment given valid comment_id', () => {
+        const deletedComment = {
+            comment_id: 6,
+            body: "I hate streaming eyes even more",
+            votes: 0,
+            author: "icellusedkars",
+            article_id: 1,
+            created_at: '2020-04-11T21:02:00.000Z',
+          }
+        return request(app)
+        .delete('/api/comments/6')
+        .expect(200)
+        .then(({body}) => {
+            expect(body).toEqual(deletedComment)
+        })
+    })
```

Comment:
For delete comments, you would expect a status 204 that responds with no content. How would you write a test that expects nothing back? 🤔